docs/teachertool/catalog.json

```
@@ -12,6 +12,112 @@
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "162222FB-8E2B-4131-9EA3-EC6B83DDAFB2",
+            "use": "row_variable_set_random",
+            "template": "Variable named 'row' is set to a random value",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "01A65046-D4BE-44B6-8273-7A878A71B3D0",
+            "use": "point_bool_check",
+            "template": "Check that an LED is lit on the screen",
+            "description": "At the spot 'col', 'row' on the LED screen, the LED is on",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "E3286B6D-5BEB-43EC-B246-C956AAC34C3E",
+            "use": "unplot_vars_used",
+            "template": "Unplot a point on the LED screen at position 'col', 'row'",
+            "description": "Uses the block that turns off an LED",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "46AFEDA3-35CB-4041-BEC4-53C0D29E57BA",
+            "use": "col_add_num",
+            "template": "Sum the value of the 'col' variable and a given number",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "6BA59761-9464-4C8D-9D8F-E8985273FB92",
+            "use": "plot_vars_used",
+            "template": "Plot a point on the LED screen at position 'col' + 1, 'row'",
+            "description": "Uses the block that turns on an LED",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "1B331451-081E-4F26-9CB6-F5A649432BF9",
+            "use": "repeat_loop_used",
+            "template": "Repeat loop is used",
+            "description": "A repeat loop is found in the program",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "A4F05FAB-72F2-48F5-8E33-7AE2CBB6CD43",
+            "use": "forever_used",
+            "template": "Forever loop is used",
+            "description": "A forever loop is found in the program",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "5934E4C6-7AE4-46A4-8F92-99101987D064",
+            "use": "show_icon_on_start",
+            "template": "Show an icon on the LED screen when the program starts",
+            "description": "An icon block is used inside the on start block",
+            "hideInCatalog": true,
+            "docPath": "/teachertool"
+        },
+        {
+            "id": "0B2BC680-D79D-42C8-BA13-B6B189E2BF7D",
+            "use": "point_condition",
+            "template": "If an LED point is truthy, then uplot that point and plot a new point",
+            "description": "An if statment checks that a specific LED is on/1/true. If this is true, turn that light off and turn on a new light",
```

```suggestion
            "description": "An if statment checks that a specific LED is lit, turns that light off, and then turns on a new light",
```

---

I'm totally fine with changing it, this is one spot in particular that I chatted with Kiki about, though. She agreed that describing the LED as lit would be a sound way forward, but also proposed that we could use the opportunity to teach through context here. By saying "we're checking that a specific LED is on/1/true", we're able to explain what it means to be truthy. 

Again, totally fine with changing it, but I wanted to present it in that light before doing so.

---

Personally, I don't think this is the place to try and teach teachers this sort of thing. We could put something like that in the curriculum itself, but the goal of this tool is to simplify their process and reduce cognitive load. This adds to it (quite significantly).

---

spelling: statment -> statement

---

Referring to an LED as a "light" seems inconsistent here, It would also conflict with the usage of that word in my suggestion above, should you choose to accept it. Maybe:

```suggestion
            "description": "Checks to see if a certain LED is on, and if so, turns it off and turns a different LED on.",
```

---

I'm thinking that we should just get rid of the description for this one. They sound too similar for the description to be valuable.